atom-feed/pom.xml

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Do you also want to conditionally load the atom-feed submodule jar in config.xml such that even BedAssignmentAdvice and BedAssignmentAdvice are never loaded when the atomfeed module is not running?

Reply:
Good question - I'm not super concerned.  If it's a better approach, we can always do that in a follow-up commit.  For now this seems to work without that added complexity.

Reply:
Fair enough. 😊